pkg/cli/cmds/k3sopts.go

```diff
@@ -94,3 +83,61 @@ func commandFromK3S(cmd *cli.Command, flagOpts map[string]*K3SFlagOption) (*cli.
 	cmd.Flags = newFlags
 	return cmd, errs.Err()
 }
+
+// flagSetUsage receives a flag and a K3S flag option, parses
+// both and sets the necessary fields based on the underlying
+// flag type.
+func flagSetUsage(flag cli.Flag, opt *K3SFlagOption) {
+	v := reflect.ValueOf(flag).Elem()
+	if v.CanSet() {
+		switch t := flag.(type) {
+		case *cli.StringFlag:
+			t.Usage = opt.Usage
+		case *cli.StringSliceFlag:
+			t.Usage = opt.Usage
+		case *cli.BoolFlag:
+			t.Usage = opt.Usage
+		}
+	}
+}
+
+// flagSetDefault receives a flag and a K3S flag option, parses
+// both and sets the necessary fields based on the underlying
+// flag type.
+func flagSetDefault(flag cli.Flag, opt *K3SFlagOption) {
+	v := reflect.ValueOf(flag).Elem()
+	if v.CanSet() {
+		switch t := flag.(type) {
+		case *cli.StringFlag:
+			x := opt.Default
```

Comment:
Do we have to create a new variable for the destination vs just passing through to the upstream destination, or is there some other reason (such as destination being nil by default)?